File: hotel_search/v2/by_city/ios/HotelSearch.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// Get list of hotels by city code
amadeus.shopping.hotelOffers.get(params: ["cityCode": "PAR"],
                                 onCompletion: {
                                     response, error in
                                     if error == nil {
                                         print(response!.data)
                                     }
                                        })

File: hotel_search/v2/get/hotel_offers/Python SDK/hotel_search.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    # Get list of Hotels by city code
    hotels_by_city = amadeus.shopping.hotel_offers.get(cityCode='PAR')
    print(hotels_by_city.data)
except ResponseError as error:
    raise error
